data/variable/database/templates/users.mysql.sql

```diff
@@ -2,7 +2,7 @@ CREATE TABLE users (
 
     id       INTEGER UNSIGNED NOT NULL PRIMARY KEY AUTO_INCREMENT,
     username VARCHAR(50),
-    digesta1 VARCHAR(32),
+    digesta1 VARCHAR(60),
```

Comment:
I think for this one we should make the exception, because `digesta1` has a very specific meaning: the `HA1` component of a digest hash, and we are kind of abusing the field to mean something it's not =)